.gitattributes

```diff
@@ -1,7 +1,10 @@
-# default
+# Set the default behavior, in case people don't have `core.autocrlf` set.
 * text=auto
 
-# Windows bat files in mobile/android
+# Declare files that will always have LF line endings on checkout.
+*.php text eol=lf
```

Comment:
Can you explain this particular change. It isn't clear why PHP files would need special treatment. It also seems completely unrelated to the Husky upgrade.

Reply:
@gziolo This is useful for users on Windows OS who don't have `core.autocrlf` set. Yeah, it seems a bit unrelated to the Husky upgrade but I have attached this small change as I figured out while touching the PHP files on Gutenberg. More over this was derived from the WooCommerce project with this raised PR https://github.com/woocommerce/woocommerce/pull/27456

Reply:
I would prefer to have this change proposed as a separate PR. I tested everything else and it works as expected.